tests/modules/nf-core/snakemake/main.nf.test

```diff
@@ -41,7 +41,7 @@ rule hello_world:
 
         then {
             assert process.success
```

Comment:
recommend enclosing all asserts within `assertAll()` so the tests don't stop at the first fail